grakn-engine/src/main/java/ai/grakn/engine/postprocessing/UpdatingInstanceCountTask.java

```diff
@@ -55,7 +55,7 @@
 
     @Override
     public boolean start() {
-        final long shardingThreshold = engineConfiguration().getPropertyAsLong(AbstractGraknGraph.SHARDING_THRESHOLD);
+        final long shardingThreshold = engineConfiguration().getPropertyAsLong(GraknTxAbstract.SHARDING_THRESHOLD);
```

Comment:
Honestly for ease of "remembering" what the hell is the name of the factory or graph of whatever. When using intelliJ I often do Shift-Shift followed by `GJGF` of `GFJ` when I am desperately trying to find the GrkanJanus thing. This makes is easier because I can just do GraTx.... and all the options will appear. 

I.e. Filipe dumb, autocomplete smart.